internal/kubelet/client.go

```diff
@@ -124,6 +128,9 @@ func (p *kubeConfigClientProvider) BuildClient() (Client, error) {
 	}
 
 	joinPath, err := url.JoinPath(authConf.Host, "/api/v1/nodes/", p.endpoint, "/proxy/")
+	if p.cfg.CAFile != "" {
+		fmt.Printf("Kubeconfig Client Provider will use custom CA cert: %v\n", p.cfg.CAFile)
```

Comment:
If you're going to log, use the logger.
```suggestion
		p.logger.Debug(fmt.Sprintf("Kubeconfig Client Provider will use custom CA cert: %v\n", p.cfg.CAFile))
```